src/adaptors/adaptor_common.h

```diff
@@ -86,14 +86,11 @@ bool qd_tls_initial_setup(qd_adaptor_config_t *config,
 
 /**
  * Grants as many read qd_adaptor buffers as returned by pn_raw_connection_read_buffers_capacity() with a maximum of
- * RAW_BUFFER_BATCH(16) read buffers. Stuffs the granted adaptor buffers into the passed in granted_read_buffs list if
- * provided.
+ * RAW_BUFFER_BATCH(16) read buffers.
```

Comment:
> with a maximum of RAW_BUFFER_BATCH(16) read buffers.

Really? Reading the code that doesn't seem to be the case - it will always grant read_buffers_capacity().  It is not limited to RAW_BUFFER_BATCH - it simply passes at most RAW_BUFFER_BATCH buffers in one call to pn_raw_connection_give_read_buffers. The outer loop calls pn_raw_connection_give_read_buffers() multiple times to fill the entire capacity.

Which is correct - it should fill the capacity. My point is that it appears that the comment is wrong.

Reply:
I fixed the comment.